docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -6,7 +6,9 @@ v0.11.2 (Anticipated December, 2024)
 
 Deprecations
 ~~~~~~~~~~~~
-
+* Changed the `g_poa_effective` parameter name to
+  `effective_irradiance` in :py:func:`~pvlib.pvsystem.PVSystem.pvwatts_dc`.
+  (:pull:`2235`)
```

Comment:
Maybe it's still worth linking the related issue?  I understand they aren't directly connected though. Just a thought
```suggestion
  (:issue:`1253`, :pull:`2235`)
```